messages/run-dfa.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
 		"rulethreadtimeoutDescriptionLong": "Time limit for evaluating a single entrypoint. Value in milliseconds. Inherits from SFGE_RULE_THREAD_TIMEOUT env-var if set. Default is 900,000 ms, or 15 minutes.",
 		"sevthresholdDescription": "throws an error when violations of specific severity (or more severe) are detected, invokes --normalize-severity",
 		"sevthresholdDescriptionLong": "Throws an error if violations are found with equal or greater severity than provided value. Values are 1 (high), 2 (moderate), and 3 (low). Exit code is the most severe violation. Using this flag also invokes the --normalize-severity flag",
+		"sfgejvmargsDescription": "JVM arguments to optimize SFGE execution to your system",
+		"sfgejvmargsDescriptionLong": "Specify JVM arguments to override system defaults while executing SFGE. For multiple arguments, add them to the same string separated by space.",
```

Comment:
@teresa-allen-sfdc Here's a new parameter to dfa command - could you include this in your review?